openwisp_radius/tests/test_api/test_rest_token.py

```diff
@@ -81,15 +82,6 @@ def test_user_auth_token_400_credentials(self):
             'credentials entered are not valid', r.json()['non_field_errors'][0]
         )
 
-    @capture_any_output()
-    def test_user_auth_token_400_organization(self):
-        url = self._get_url()
-        self._get_org_user()
-        OrganizationUser.objects.all().delete()
-        r = self.client.post(url, {'username': 'tester', 'password': 'tester'})
-        self.assertEqual(r.status_code, 400)
-        self.assertIn('is not member', r.json()['non_field_errors'][0])
-
```

Comment:
I'd keep the test for the case in which the user has no org, just in case, but you can add this as a subtest in the new test below